src/components/organisms/ChatSidebar/components/VenueChat/VenueChat.tsx

```diff
@@ -0,0 +1,99 @@
+import React, { useEffect, useState, useMemo, FC, useCallback } from "react";
```

Comment:
I expect this component may be able to be simplified using `src/components/molecules/ChatsList/ChatsList.tsx`